README.rst

```diff
@@ -299,7 +299,7 @@ Install the required packages:
 
     cd ./django-money/
 
-    pip install -e .[test] # installation with required packages for testing
+    pip install -e ".[test]" # installation with required packages for testing
```

Comment:
Thanks, haha, this has been annoying far too long - have seen this instruction missing the quotes in many other projects.

Reply:
Can I mark this conversation as resolved?